client/my-sites/site-settings/section-import.jsx

```diff
@@ -153,8 +155,13 @@ class SiteSettingsImport extends Component {
 								<MediumImporter { ...{ key, site, importerStatus } } />
 							) ) }
 
+						{ isEnabled( 'manage/import/site-importer' ) &&
+							siteImporterImports.map( ( importerStatus, key ) => (
```

Comment:
Here, and inn other mappings I see the index used as a key - Is this not something we should be concerned about? I know that generally folk see it as an anti-pattern ([#](https://www.menubar.io/react-keys-index/) [#](https://medium.com/@robinpokorny/index-as-a-key-is-an-anti-pattern-e0349aece318) and the official [react docs](https://reactjs.org/docs/lists-and-keys.html) mention that index should only be used as a last resort. I'm not so sure if there is anything better in the data though so please ignore if the index is the only differentiation we can make here :)

Reply:
Yes, this is part of the legacy code :) We'll iterate and refactor the section to follow more recent practices and make it work/look better.